.github/workflows/CI.yml

```diff
@@ -0,0 +1,44 @@
+name: CI
+on:
+  push:
+      paths-ignore:
+        - 'README.md'
+      branches:
+        - main
+        - master
+  pull_request:
+      paths-ignore:
+        - 'README.md'
+      branches:
+        - main
+        - master
+jobs:
+  test:
+    name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - '1'
+          - '1.6'
+          - 'nightly'
+        os:
+          - ubuntu-latest
+        arch:
+          - x64
+          # - x86
```

Comment:
why not test on 32-bit?

Reply:
```suggestion
          - x86
```

Reply:
no good reason, I just had it disabled while I was getting everything working